jmix-quartz/quartz-flowui/src/main/java/io/jmix/quartzflowui/view/jobs/JobModelListView.java

```diff
@@ -129,16 +142,71 @@ private String getHeaderForPropertyColumn(String propertyName) {
         return messageTools.getPropertyCaption(jobModelsDc.getEntityMetaClass(), propertyName);
     }
 
+    protected void initDataChangedEventListener() {
+        VaadinSession session = VaadinSession.getCurrent();
+        if (session == null) {
+            return;
+        }
+
+        UiEventsManager uiEventsManager = session.getAttribute(UiEventsManager.class);
+        if (uiEventsManager != null) {
+            uiEventsManager.addApplicationListener(this, this::onApplicationEvent);
+
+            // Remove on detach event
+            addDetachListener(event -> uiEventsManager.removeApplicationListeners(this));
+        }
+    }
+
+    protected void onApplicationEvent(ApplicationEvent event) {
+        if (event instanceof QuartzDataChangedEvent dataChangedEvent) {
+            onDataChangedEvent(dataChangedEvent);
+        }
+    }
+
+    protected void onDataChangedEvent(QuartzDataChangedEvent event) {
+        loadJobsData();
+    }
+
     @Subscribe
     protected void onBeforeShow(View.BeforeShowEvent event) {
         nameFilter.addTypedValueChangeListener(this::onFilterFieldValueChange);
         classFilter.addTypedValueChangeListener(this::onFilterFieldValueChange);
         groupFilter.addTypedValueChangeListener(this::onFilterFieldValueChange);
         jobStateFilter.addValueChangeListener(this::onFilterFieldValueChange);
 
+        setupQuartzDataChangedListener();
+
         loadJobsData();
     }
 
+    protected void setupQuartzDataChangedListener() {
+        quartzService.addTriggerListener(new TriggerListener() {
```

Comment:
Every time someone open this view new listener is added to scheduler via QuartzService.

- We need one listener for quartz events - it should transform incoming quartz events into new start\finish events and publish them via UI publisher
  - Use JobSupportListener instead of TriggerListener (https://docs.jmix.io/jmix/quartz/index.html#track-job-execution)
  - Create it as a separate bean
  - Make it self-registered
- View should just listen for start\finish application events

Reply:
done